test/parallel/test-buffer.js

```diff
@@ -289,8 +291,25 @@ for (var i = 0; i < asciiString.length; i++) {
   assert.equal(sliceA[i], sliceB[i]);
 }
 
-// TODO utf8 slice tests
+// UTF-8 slice test
+
+var utf8String = '¡hèlló wôrld!';
```

Comment:
These characters are all within the latin-1 charset. check via
```js
Buffer('¡hèlló wôrld!','binary').toString('binary') === '¡hèlló wôrld!'
```
Mind using a two byte utf-8 character?